snare

```diff
@@ -0,0 +1,72 @@
+#! /usr/bin/env python3.7
+
+# Copyright (c) 2019 King's College London, created by the Software Development
+# Team <http://soft-dev.org/>
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0>, or the MIT license <LICENSE-MIT
+# or http://opensource.org/licenses/MIT>, or the UPL-1.0 license
+# <http://opensource.org/licenses/UPL> at your option. This file may not be
+# copied, modified, or distributed except according to those terms.
+
+import getopt, hashlib, hmac, json, os, subprocess, sys, tempfile, traceback
```

Comment:
flake8 found:

```
snare:12:1: F401 'traceback' imported but unused
```

It also found a load of code style nits (long lines, whitespace etc), but I guess you are not doing PEP8.

Reply:
Ah, yes, I had removed the need for `traceback`. Will remove the import.